### PR TITLE
Updates 'sheerlike' and makes a small tweak to BASE_SETTINGS

### PR DESCRIPTION
**Pull request by rosskarchner**

### CODE REVIEW

**Comment by kave** [comment]
:+1:

**Comment by kurtw** [comment]
lgtm

File: requirements/base.txt [hunk 1]
```diff
@@ -1,10 +1,10 @@
 unipath>=1.1,<=2.0
 django>=1.8,<=1.9
-git+https://github.com/cfpb/django-sheerlike.git#egg=sheerlike
+git+https://github.com/cfpb/django-sheerlike.git@2d71ef1499ebdb420803abcda8d54397d37a1b2e#egg=sheerlike
```
Comment (kurtw) [inline requirements/base.txt]: The sheerlike PR got pulled in so this isn't necessary anymore.

**Comment by rosskarchner** [comment]
fixed that @kurtw

**Pull request merged**